File: zero/Maruko.Zero/Internal/Domain/SysMenu.cs
using System.ComponentModel.DataAnnotations;
using FreeSql.DataAnnotations;
using Maruko.Core.FreeSql.Internal;

namespace Maruko.Zero
{
    [Table(Name = "sys_menu")]
    public class SysMenu : FreeSqlEntity
    {
        [Column(Name = "parentId")]
        public int ParentId { get; set; } = 0;

        [Column(Name = "name", StringLength = 20)]
        public string Name { get; set; }

        [Column(Name = "url", StringLength = 100)]
        public string Url { get; set; }


        [Column(DbType = "tinyint(4)", Name = "level")]
        public int Level { get; set; } = 1;


        [Column(Name = "operates", StringLength = 100)]
        public string Operates { get; set; }

        public int Sort { get; set; }

        [Column(Name = "icon", StringLength = 100)]
        public string Icon { get; set; }

        [Column(Name = "key", StringLength = 50)]
        public string Key { get; set; }

        public int AddOperateSort()
        {
            Sort += 1;
            return Sort;
        }
    }
}

File: act/act-modules/act-modules-zero/src/main/java/com/act/modules/zero/internal/domain/SysMenu.java
package com.act.modules.zero.internal.domain;

import com.act.core.domain.BaseEntity;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 *
 */
@Data
@TableName("sys_menu")
@SuppressWarnings("all")
public class SysMenu extends BaseEntity<Long> {
    @TableField("parentId")
    private Long parentId;

    @TableField("name")
    private String name;

    @TableField("url")
    private String url;

    @TableField("level")
    private int level = 0;

    @TableField("operates")
    private String operates;

    @TableField("sort")
    private int sort = 0;

    @TableField("icon")
    private String icon;

    @TableField("`key`")
    private String key;

    @TableField("isLeftShow")
    private int isLeftShow = 0;

    public int AddOperateSort() {
        sort += 1;
        return sort;
    }
}
